secret/native/create_test.go

```diff
@@ -46,6 +48,9 @@ func TestNative_Create_Org(t *testing.T) {
 		t.Errorf("Create returned err: %v", err)
 	}
 
+	value, _ := decrypt([]byte(want.GetValue()), passphrase)
+	want.Value = &value
```

Comment:
true for some other spots, too.. but maybe use the setter, ie. `want.SetValue(value)`

Reply:
I think found them all https://github.com/go-vela/server/pull/212/commits/1167fb6c4c43f36914c2630011740138ce9615e1